### ISSUE TITLE
Inferior location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11558]
Preferred term label:
Inferior location

Synonyms

    Lower location

    Inferiorly located

    Caudal location

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization referring to a position below or lower than another structure. In ophthalmology, it is frequently used to describe inferior retinal colobomas, inferior visual field defects, or inferior corneal punctate epitheliopathy. PMID:22384005

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Uveal coloboma (OMIM:120200) (typically located inferiorly due to embryonic fissure closure)

    Blepharokeratoconjunctivitis (Orphanet:536511)

Your nano-attribution (ORCID)
0000-0001-9042-0508
